package.json

```diff
@@ -8,7 +8,11 @@
     "start": "next start",
     "lint": "next lint",
     "prepare": "husky",
-    "fmt": "prettier --write ."
+    "fmt": "prettier --write .",
+    "docker-build-dev": "docker build -t dibbs-site:dev .",
+    "docker-run-dev": "docker run -p 3000:3000 dibbs-site:dev",
```

Comment:
We can remove these since we got rid of the dev image.

Reply:
done